src/components/manifold-plan-table/manifold-plan-table.tsx

```diff
@@ -440,19 +440,19 @@ export class ManifoldPlanTable {
 
               // undefined / disabled feature
               return (
-                <div class="mp--cell mp--cell__body">
-                  <span class="mp--empty-cell">•</span>
+                <div class="ManifoldPlanTable__Cell ManifoldPlanTable__Cell--Body">
+                  <span class="ManifoldPlanTable__Cell__Disabled">•</span>
                 </div>
               );
             }),
             <div
-              class="mp--cell mp--cell__body mp--cell__bbs"
+              class="ManifoldPlanTable__Cell ManifoldPlanTable__Cell--Body"
               data-row-last
               data-column-last={lastColumn}
             >
               <a
-                data-cta="cta-button"
-                class="mp--button"
+                data-testid="cta"
+                class="ManifoldPlanTable__Button"
```

Comment:
cc @jeffandersen no rush but wanted to see how you felt: We decided on prefixing all CSS class names with the element name, since that has to be unique. It’s a good way for us to prevent style conflicts with the providers’ styles, as well as our own.

So to style a `button` on `<manifold-plan-table>`, you target `.ManifoldPlanTable__Button`. Predictable, right? We’re using BEM which should be familiar-ish to many.

Anyway the story works technically, but curious if you have any opinions on the naming / namespacing.